index.js

```diff
@@ -59,69 +60,55 @@ is.string = x => typeof x === 'string';
 is.number = x => typeof x === 'number';
 is.boolean = x => typeof x === 'boolean';
 is.symbol = x => typeof x === 'symbol';
+is.nullOrUndefined = x => is.null(x) || is.undefined(x);
```

Comment:
Don't change the order. The order of defined methods is deliberate and should be the same as in the docs.

Reply:
So why would then nullOrUndefined be separated from other primitive predicates?

I accept the review, but I would still like an explanation, from the user perspective, not the dev.

Reply:
Because it's not a pure type check, it's a combinatory check, and therefore is better placed in Misc. Same as the `primitive` and `typedArray` check. Happy to reconsider if you have some good arguments against that, though.

Reply:
I do understand now, what I had in mind in the moment is that because these are `primitive` related methods, then they should be placed alongside all other `primitive` methods.

also, I wanted to use it in a method that comes before where `nullOrUndefined` is originally placed.

Same for `typedArray`, there are several specific checks for all the different `typedArray`s, so it'd be logical if it was placed next to it.